### PR TITLE
Add Swagger for NDLA endpoints

### PR DESCRIPTION
**Pull request by emmachughes**

### CODE REVIEW

File: sourcecode/hub/tests/Browser/NdlaTest.php [hunk 1]
```diff
@@ -0,0 +1,20 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Browser;
```
Comment (chrieinv) [inline sourcecode/hub/tests/Browser/NdlaTest.php:5]: Incorrect namespace, should be `Tests\Browser`

File: sourcecode/hub/resources/schema/ndla-openapi.json [hunk 2]
```diff
@@ -0,0 +1,366 @@
...
+                "schema": {
+                    "type": "string",
+                    "enum": ["true", "false"]
+                }
+            },
+
+            "locale": {
+                "name": "locale",
+                "in": "query",
+                "description": "Locale to launch in",
```
Comment (chrieinv) [inline sourcecode/hub/resources/schema/ndla-openapi.json:34]: Include locale format or example(s)

File: sourcecode/hub/tests/Browser/NdlaTest.php (hunk 1 above)
Comment (emmachughes) [reply sourcecode/hub/tests/Browser/NdlaTest.php:5]: PhpStorm keeps doing this. Maybe we need a CI job that fails when classes aren't namespaced correctly.

Comment (chrieinv) [reply sourcecode/hub/tests/Browser/NdlaTest.php:5]: Maybe the namespaces aren't configured correctly in PhpStorm, you can check by going to Settings and Directories

Comment (chrieinv) [reply sourcecode/hub/tests/Browser/NdlaTest.php:5]: Also make sure the `For generated source` is checked

**Review by chrieinv — APPROVED** [review]

**Pull request merged**